_install/synology.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: documentation
+title:  Synology NAS hosting
+date:   2021-01-10
+order:  99
+---
+
+This procedure if for Synology NAS running DiskStation Manager (DSM) version 6.X.
+
+# Pre-requisites
+
+## Packages Installation
+
+Open the DiskStation Manager interface. Use the Package Center to install the following packages (note they may already be installed):
+* Web Station
+* MariaDB 10
+* phpMyAdmin (only required for initial set-up of the database, may be removed once Webtrees is running).
+
+## Webstation configuration
+
+1. In DSM, open "Web Station".
+2. Click on "General Setting" and check which PHP profile is being used ("Default PHP 7.4" in the below example):
+
+   ![Click on General Settings, check PHP profile.](../assets/img/synology/WS-GeneralSettings.png)
+
+3. Click on "PHP Settings" then select the the PHP profile used by Web Station:
+
+   ![Click on PHP Settings, select default PHP profile.](../assets/img/synology/WS-PHPSettings.png)
+
+4. Edit the selected PHP profile, make sure all extensions are activated in the Extensions list. Click OK:
```

Comment:
All extensions?  Does this include `xdebug`.

For performance, you should not install this...

Reply:
Synology NAS are supposedly production only environments so xdebug is not part of the available extensions.

Reply:
OK fine.  I cannot test this - but I googled xdebug/synology and saw a comment that said it could be enabled via the control panel...

Reply:
It can, but not in this list. As shown in the screen capture, enabling xdebug or display_error is in the upper part of the setting window (see https://github.com/yann64/webtrees.github.io/blob/master/assets/img/synology/WS-PHPProfileSettings.png ). In theory the "Extensions" list is only for "production friendly" and can all be activated.
That being said you have a good point and I will edit the doc to mention neither xdebug nor display_error should be activated.